### ISSUE TITLE
SpEffectParam Export->Import Produces Strange Behavior Ingame

### ISSUE THREAD

**Issue by coldpaintedash** [vawser/Yapped-Rune-Bear#44]
In the latest version of the program simply exporting and then re-importing SpEffectParam causes the Greatshield Talisman to give seemingly infinite guard boost to two-handed weapons (and maybe other things; I didn't test them all).

**Comment by vawser** [comment]
I'm not seeing anything wrong with the data. If you have older data (either via the CSV file or an older regulation.bin), the "Guard Stability %" field may still have the "1.1" value it had before the 1.04 update, where FROM changed it to "1". 

That would explain the guard boost change, since I believe FROM has embedded the guard boost effect within the state info field, so essentially the effect is double dipping.

**Comment by coldpaintedash** [comment]
No, it's not the data. The "Guard Stability %" field is still 1 after I go through to whole process. I'm using a clean regulation.bin (freshly validated from steam), a clean install of Yapped 2.1.0, and no CSV files initially present. All I did was export and immediately re-import SpEffectParam.
